docs/config.md

```diff
@@ -5,8 +5,7 @@
 | Environment variable | Description | Default value |
 |-|-|-|
 | `OTEL_DOTNET_AUTO_HOME` | Installation location. |  |
-| `OTEL_DOTNET_AUTO_TRACES_ENABLED` | Enables the tracer. | `true` |
-| `OTEL_DOTNET_AUTO_METRICS_ENABLED` | Enables the meter. | `true` |
+| `OTEL_DOTNET_AUTO_ENABLED` | Enables CLR profiler needed by Tracer. | `true` |
```

Comment:
👍

Reply:
I'm wondering if this setting should also affect either the startup hook or the code that attempts to configure the SDK? The name implies that it should affect everything, but the description states that it only affects the profiler (leading to a difference in behavior between .net framework and .net core 3.1+).

Reply:
The setting only affects the CLR profiler: it makes it return E_FAIL on the initialize call, which means no further calls to the CLR profiler.

Reply:
Right. so what I'm referring to is that for .net framework apps, that will lead to no auto instrumentation running at all. However, for a .net 6 app, because we don't rely on the clr profiler, auto instrumentation will still be available (for source code instrumentation). We may want to at least call out that difference in behavior.

Reply:
@nrcventura, what do you think about following statement? If you are fine please please apply it to PR.

```suggestion
| `OTEL_DOTNET_AUTO_ENABLED` | Enables CLR profiler. If `false`, it disables Automatic Instrumentation for .NET Framework and it disables binary instrumentation for .NET (Core). | `true` |
```

Reply:
I find `binary instrumentation` a bit confusing, elsewhere we used `bytecode instrumentation` iirc.

Reply:
I like the combination of suggestions from @Kielek and @pjanotti.